crates/typst/src/symbols/sym.rs

```diff
@@ -126,6 +126,8 @@ pub(crate) const SYM: &[(&str, Symbol)] = symbols! {
     hyph: ['‐', minus: '\u{2D}', nobreak: '\u{2011}', point: '‧', soft: '\u{ad}'],
     percent: '%',
     copyright: ['©', sound: '℗'],
+    copyleft: '🄯',
+    trademark: ['™', service: '℠', registered: '®'],
```

Comment:
trademark might be better suited next to servicemark maybe?

Reply:
copyright and copyleft as well probably

Reply:
I had not realized `servicemark` already existed.

Reply:
Actually I hadn't realized that you added `trademark.service`. Maybe it's a better name than `servicemark`...